ui/app/templates/access-control/policies/policy.hbs

```diff
@@ -12,17 +12,9 @@
 		</div>
 		{{#if (can "destroy policy")}}
 			<div>
-				<TwoStepButton
-					data-test-delete-button
-					@idleText="Delete policy"
-					@cancelText="Cancel"
-					@confirmText="Yes, delete"
-					@confirmationMessage="Are you sure?"
-					@awaitingConfirmation={{this.deletePolicy.isRunning}}
-					@onConfirm={{perform this.deletePolicy}}
-					@onPrompt={{this.onDeletePrompt}}
-					@onCancel={{this.onDeleteCancel}}
-				/>
+        <Hds::Button @text="Delete Policy" @color="critical"
+          {{on "click" (perform this.deletePolicy)}}
+        />
```

Comment:
In updating to user our new design systems buttons, one trade-off is that two-step buttons (where you first click something, then get an "are you sure? yes/cancel" prompt in its place) aren't supported.

This change has led to a lot of discussion internally, and I will probably try to introduce a two-step button pattern using the new design system soon. For now, however, former "are you sure?" buttons are just a bit more final.